index.html

```diff
@@ -293,62 +293,70 @@
 reminder of their humanity. When we use the term [=user=], it is to talk about the specific [=person=]
 who happens to be using a given system at that time.
 
-We define <dfn data-lt="data">personal data</dfn> as any information relating to a [=person=] such
-that:
-
-* this [=person=] is identified, directly or indirectly, by reference to an
-  identifier such as a name, email address, an arbitrary identifier or
-  identification number, an online identifier such as an IP address or any
-  identifier attached to a device this [=person=] may be using, phone number,
-  location data, or factors specific to the physical, physiological, genetic,
-  mental, economic, cultural, or social identity of that [=person=], as well as
-  identifiers derived from such data, for instance through hashing; or
-* this [=person=] could reasonably be reidentified from a conjunction of this
-  data with other data; or
-* the data pertains to a group of people such that a [=person=] may find
-  themselves to be the subject of a treatment related to this group, even if the
-  entity carrying out the treatment has no way to identify that [=person=].
+We define <dfn data-lt="data">personal data</dfn> as any information relating to a [=person=],
+identified or identifiable, directly or indirectly, in particular by reference to an [=identifier=].
+
+[=Data=] is personal if this [=person=] could reasonably be reidentified from a conjunction of this
+data with other data.
 
 Data is <dfn>permanently de-identified</dfn> when there exists a high level of confidence
```

Comment:
Re @sandandsnow's [comment](https://github.com/w3ctag/privacy-principles/pull/30#issuecomment-970797540), Google calls this category "anonymous data". That covers both k-anonymity and differential privacy-based anonymization techniques, and probably others, but DP tends to be preferred. It's probably fine to avoid talking about techniques in this document?